src/test_l1_dsVideoDevice.c

```diff
@@ -1765,15 +1803,15 @@ void test_l1_dsVideoDevice_negative_dsRegisterFrameratePostChangeCB(void)
 
     // Step 02: Initialize video devices
     result = dsVideoDeviceInit();
-    UT_ASSERT_EQUAL(result, dsERR_NONE);
+    UT_ASSERT_EQUAL_FATAL(result, dsERR_NONE);
 
     // Step 03: Register with invalid callback (NULL)
     result = dsRegisterFrameratePostChangeCB(NULL);
     UT_ASSERT_EQUAL(result, dsERR_INVALID_PARAM);
 
     // Step 04: De-initialize the video devices
     result = dsVideoDeviceTerm();
-    UT_ASSERT_EQUAL(result, dsERR_NONE);
+    UT_ASSERT_EQUAL_FATAL(result, dsERR_NONE);
```

Comment:
in line :1835 remove the pSuite2 and combine with pSuite1 
pSuite2 = UT_add_suite( "[L1 dsVideoDevice - advanced]", NULL, NULL );

Reply:
done